arcgis-ios-sdk-samples/AppDelegate.swift

```diff
@@ -169,9 +177,9 @@ class AppDelegate: UIResponder, UIApplicationDelegate, UISplitViewControllerDele
 }
 
 extension UIColor {
-    class var primaryBlue: UIColor { return #colorLiteral(red: 0, green: 0.475, blue: 0.757, alpha: 1) }
-    class var secondaryBlue: UIColor { return #colorLiteral(red: 0, green: 0.368, blue: 0.584, alpha: 1) }
-    class var backgroundGray: UIColor { return #colorLiteral(red: 0.973, green: 0.973, blue: 0.973, alpha: 1) }
-    class var primaryTextColor: UIColor { return #colorLiteral(red: 0.196, green: 0.196, blue: 0.196, alpha: 1) }
-    class var secondaryTextColor: UIColor { return #colorLiteral(red: 0.349, green: 0.349, blue: 0.349, alpha: 1) }
+    // Also used as global tint/accent color.
+    class var accentColor: UIColor { return UIColor(named: "AccentColor")! }
+    class var backgroundGray: UIColor { return UIColor(named: "backgroundGray")! }
```

Comment:
A couple comments about this color:

- It is defined using a system color that was introduced in iOS 13. What is the behavior on iOS 12? I'm a bit surprised there's not a warning about that.
- It is only used in two places. Could we changes those places to use a system color and then get rid of this color?

Reply:
> * It is defined using a system color that was introduced in iOS 13. What is the behavior on iOS 12? I'm a bit surprised there's not a warning about that.

My guess is it uses a similar strategy as storyboard. When a system color is assigned, storyboard will provide a fallback color for iOS versions prior to 13 (e.g. white color is backgroundColor in light mode, in iOS 12).
```xml
<color key="backgroundColor" systemColor="systemBackgroundColor" cocoaTouchSystemColor="whiteColor"/>
```
Though I cannot tell from its content json...

> * It is only used in two places. Could we changes those places to use a system color and then get rid of this color?

Yes. Let me figure it out...

Reply:
It can be replaced by either `secondarySystemBackgroundColor` or `systemGray6`, but both colors are iOS 13+ only. Since we want to make this color Dark Mode adaptive, we can either keep this custom named color in asset catalog, or add it via code with `if #available iOS 13`. I tend to keep it in the assets. What do you think?